### PR TITLE
Update statedump.md

### PR DESCRIPTION
**Pull request by thotz**
Statedump enhance to take statedump of gfapi clients. Hence mentioning the same

### CODE REVIEW

File: Troubleshooting/statedump.md [hunk 1]
```diff
@@ -46,7 +46,10 @@ For the NFS server:
 For quotad:
 
                 gluster volume statedump <volname> quotad
+                
+For gfapi clients: (like nfs-ganesha/samba)
 
+                gluster volume statedump <volname> client localhost:<process-id>
```
Comment (prashanthpai) [inline Troubleshooting/statedump.md]: Please use a generic placeholder such as `<host>` and describe what `host` stands for.

Comment (thotz) [reply Troubleshooting/statedump.md]: AFAIR currently it works only with localhost, that's why I have given as localhost

Comment (thotz) [reply Troubleshooting/statedump.md]: Can u please refer to this bug https://bugzilla.redhat.com/show_bug.cgi?id=1454311.
Status of bug is still new. I didn't try out on latest gluster bits

Comment (prashanthpai) [reply Troubleshooting/statedump.md]: I believe that's specific to NFS Ganesha where volfileserver is configured to be `localhost`

Snippet from bug
```
As dscussed, for Ganesha and Samba and i think  Qemu as well, localhost is what has to be  specified, as localost is the default volfileserver
```

```
I could get the gluster statedump of the gfapi application ( QEMU ) with the hostname or IP. In this case, gfapi application (QEMU) is running on the host outside of Trusted Storage Pool.
```

You change the doc to be specific about the same.

**Pull request closed without merging**